wolfhsm/wh_server.h

```diff
@@ -42,54 +73,65 @@ typedef struct whServerContext_t whServerContext;
 #include "wolfhsm/wh_nvm.h"
 #include "wolfhsm/wh_message_customcb.h"
 
-#ifndef WOLFHSM_NO_CRYPTO
+#ifndef WOLFHSM_CFG_NO_CRYPTO
 #include "wolfssl/wolfcrypt/settings.h"
+#include "wolfssl/wolfcrypt/types.h"
 #include "wolfssl/wolfcrypt/random.h"
 #include "wolfssl/wolfcrypt/rsa.h"
 #include "wolfssl/wolfcrypt/ecc.h"
 #include "wolfssl/wolfcrypt/curve25519.h"
 #include "wolfssl/wolfcrypt/cryptocb.h"
-#endif /* WOLFHSM_NO_CRYPTO */
+#endif /* WOLFHSM_CFG_NO_CRYPTO */
 
-#ifdef WOLFHSM_SHE_EXTENSION
+#ifdef WOLFHSM_CFG_SHE_EXTENSION
 #include "wolfhsm/wh_she_common.h"
 #include "wolfhsm/wh_server_she.h"
 #endif
 
 /** Default server resource configurations */
-/* Number of RAM keys */
-#ifndef WOLFHSM_NUM_RAMKEYS
-#define WOLFHSM_NUM_RAMKEYS  8
+/* Reported version string */
+#ifndef WOLFHSM_CFG_INFOVERSION
+#define WOLFHSM_CFG_INFOVERSION "01.01.01"
+#endif
+
+/* Reported build identifier string */
+#ifndef WOLFHSM_CFG_INFOBUILD
+#define WOLFHSM_CFG_INFOBUILD "12345678"
```

Comment:
no action, just a reminder to myself for the future: Might be nice to pull versioning into a separate `wh_version.h` header for easier post-processing with tooling, automagically including the git hash into the binary when built on CI for release, etc

Reply:
amen!